### ISSUE TITLE
Missing link to original post

### ISSUE THREAD

**Issue by winks** [toddsundsted/ktistec#42]
This looks like a design decision to me, but I often see myself wanting to see the toot on the poster's home instance.

So for example if I have https://sup.f5n.org/remote/objects/19289 (non-public) which was originally published as https://mastodon.social/@wink/108194481005296329 - but I can't easily click anywhere in my local displayed post to get to the original.

Basically the same is that my other account is linked as https://sup.f5n.org/remote/actors/9829 but I can't easily get to https://mastodon.social/@wink - it's only in the hover text (I know I can click on the "person" link and there I see the original url, but it's an extra click and I lose the original url of the toot.

So I guess my questions are: why is that or what am I missing and would you be open to a PR to make both things configurable to show either local or remote? Yes, I seem to be clicking through to the people who were boosted into my timeline a lot.

**Comment by toddsundsted** [comment]
i'd be open to this.

it's worth consider how that might work with links to the authors, as well, and whether both could work consistently.

**Comment by vrthra** [comment]
A possible implementation -- it makes the "posted" text a hyperlink to the users home instance.

```
diff --git a/src/views/partials/object/content.html.slang b/src/views/partials/object/content.html.slang
index cdc38e2..e8b0eec 100644
--- a/src/views/partials/object/content.html.slang
+++ b/src/views/partials/object/content.html.slang
@@ -18,20 +18,24 @@ crystal:
       - if account
         - if author != actor
           a.user> data-turbo-frame="_top" title=actor.account_uri href=remote_actor_path(actor) = actor.display_name
-          | shared a #{type} by
+            a href=object.iri
+              | shared a #{type} by
           a.user< data-turbo-frame="_top" title=author.account_uri href=remote_actor_path(author) = author.display_name
         - else
           a.user> data-turbo-frame="_top" title=author.account_uri href=remote_actor_path(author) = author.display_name
-          | posted
+            a href=object.iri
+              | posted
         a.date data-turbo-frame="_top" title=object.display_date(timezone) href=remote_object_path(object) = object.short_date(timezone)
       - else
         - if author != actor
           a.user> data-turbo-frame="_top" title=actor.account_uri href=actor.display_link = actor.display_name
-          | shared a #{type} by
+            a href=object.iri
+              | shared a #{type} by
           a.user< data-turbo-frame="_top" title=author.account_uri href=author.display_link = author.display_name
         - else
           a.user> data-turbo-frame="_top" title=author.account_uri href=author.display_link = author.display_name
-          | posted
+            a href=object.iri
+              | posted
         a.date data-turbo-frame="_top" title=object.display_date(timezone) href=object.display_link = object.short_date(timezone)
   - if object.blocked?
     .extra.text
```

**Comment by toddsundsted** [comment]
@winks @vrthra this should now be fixed in 7a56cd66
